zabbixDBAdapter.py

```diff
@@ -143,7 +144,7 @@ def query_db(history_table_name, clock, cursor):
     return cursor.fetchall()
 
 
-def process_and_send_metrics(rows, latest_clock, sock=None):
+def process_and_send_metrics(rows, latest_clock, wavefront_sender=None, tags=None):
```

Comment:
I think tags can be removed and None is also not required

Reply:
Tags is a mandatory parameter that is required for the wavefront_sender.send_metric() https://github.com/wavefrontHQ/zabbix/blob/fb9fdabe8898e9e3b5fb108775da62fcd5314ed8/zabbixDBAdapter.py#L171